management/management.py

```diff
@@ -166,6 +166,53 @@ def count_unique_optimization_error_messages(
     else:
         return errors
 
+## error filtering
+
+def filter_optimizations(optimizations, to_match, return_filtered_instead=False):
+
+    if isinstance(to_match, str):
+        to_match = [to_match]
+
+    if return_filtered_instead:
+        return [opt for opt in optimizations
+                if any([tm in opt.get_error().error_message for tm in to_match])]
+    else:
+        return [opt for opt in optimizations
+                if not any([tm in opt.get_error().error_message for tm in to_match])]
+
+def filter_scf_convergence(optimizations, return_filtered_instead=False):
+
+    to_match = "Could not converge SCF iterations"
+
+    return filter_optimizations(optimizations,
+                                to_match,
+                                return_filtered_instead=return_filtered_instead)
+
+def filter_opt_convergence(optimizations, return_filtered_instead=False):
```

Comment:
This one is a little awkward, I agree we do see a lot of these but I would say that most of the time they do vanish if you look through the error cycle reports on the past few datasets we see they gradually resolve as they are restarted, so maybe it would be best to hold off on this one.

Reply:
Thanks @jthorton, good point! I've commented out this filter's usage in the list.